packages/flame_sprite_fusion/README.md

```diff
@@ -0,0 +1,45 @@
+This package adds support for parsing and rendering tilemap exported from [Sprite Fusion](https://www.spritefusion.com/)
```

Comment:
Check the other bridge packages readmes, I think they have quite similar structure to the root readme

Reply:
The contents and sections vary a lot between packages, but copied over the common header section along with the badges.